README.md

```diff
@@ -61,6 +67,76 @@ make migrate
 make collectstatic
 ```
 
+### 2. Run the server
+#### A. From PyCharm Professional
+```
+# go to deployment/ansible/development/group_vars
+cd deployment/ansible/development/group_vars
+cp all.sample.yml all.yml
+```
+
+- edit line 6 (*remote_user*), 8 (*remote_group*), and 10 (*project_path*) in all.yml accordingly
+  - make sure that *remote_user* align with your local user
+  - *remote_group* is likely not change if using linux and macOS
+  - *project_path* is equal to with your cloned directory
+
+```
+# go to deployment/ansible
+mkdir tmp
+make setup-ansible
+# choose your pycharm version from the list
+```
+
+- Open PyCharm
+- Notice your pycharm, there should be *Ford3* django server in the toolbar.
+  - Wait for a couple of minutes. Make sure the PyCharm has loaded all the necessary files.
+  - If pycharm requires to install additional supported modules, click on the provided link
+- Click on the `play button` or `debug button` next to *Ford3* instance in the toolbar. The pycharm will run the server for you
+  - sometimes restart pycharm can remedy the problem too
+  - If the server doesn't run, try:
+
+```
+# from deployment folder
+make down
+make up
+```
+
+
+#### B. From CLI
+```
+# go to deployment folder
+make migrate
```

Comment:
I also had the warning so maybe we don't need to run them at the beginning of the setup